packages/js/product-editor/src/components/header/header.tsx

```diff
@@ -63,12 +73,56 @@ export function Header( {
 			tabIndex={ -1 }
 		>
 			<div className="woocommerce-product-header__inner">
-				<div />
+				{ productType === 'product_variation' ? (
```

Comment:
Oooh I see you added the `parent_id` to the REST API, in that case could we change this condition to see if the `parent_id` exists?

For now we can just add the `product_variation` condition for going back to `tab: variations` and otherwise keep that blank so it goes to the general tab by default.